lib/user.rb

```diff
@@ -0,0 +1,13 @@
+class User
+
+  attr_reader :name
+
+    def initialize(name)
+      @name = "Sal"
+    end
+
+    def jokes
+      []
```

Comment:
close! this should be an instance variable set in `def initialize`, since it's an array that we want to keep adding to.